src/GoogleFit/FitOnFhir.GoogleFit/Resolvers/GoogleFitUserConflictResolvers.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using FitOnFhir.Common.Models;
+using FitOnFhir.GoogleFit.Client.Models;
+
+namespace FitOnFhir.GoogleFit.Resolvers
+{
+    public static class GoogleFitUserConflictResolvers
```

Comment:
nit: I wonder if it makes sense to just name these classes `GoogleFitUserExtensions` and `UserExtensions`, and just have the method names be `Merge()` (in the case of user auth `MergeForAuthorization()`). It seems like these fall into the category of extending these models, and do not require any outside data to function. When used in the code, it would look like:

```c#
await _googleFitUserTableRepository.Update(googleUser, GoogleFitUserExtensions.Merge, cancellationToken);
```

It just seems like what we are actually doing here is merging .